### PR TITLE
fix characterKey on documentation page

### PR DESCRIPTION
**Pull request by frzyc**
- Fix documentation of the `characterKey` to have all variant of the traveler, instead of `Traveler` as intended.

### CODE REVIEW

**Comment by frzyc** [comment]
Splitting the Travelers into the travelers also means that the GO exports are not fully GOOD, but since GOOD is only used as a scanner input format, this is relatively benign.

File: src/PageDocumentation/index.tsx [hunk 1]
```diff
@@ -203,7 +203,7 @@ function ArtifactSetKeyPane() {
 }
 function CharacterKeyPane() {
   const { t } = useTranslation([...new Set(allCharacterKeys)].map(k => `char_${k}_gen`))
-  const charKeysCode = `type CharacterKey\n  = ${[...new Set(allCharacterKeys)].sort().map(k => `"${k}" //${t(`char_${k}_gen:name`)}`).join(`\n  | `)}`
+  const charKeysCode = `type CharacterKey\n  = ${[...new Set(locationCharacterKeys)].sort().map(k => `"${k}" //${k === "Traveler" ? "Traveler" : t(`char_${k}_gen:name`)}`).join(`\n  | `)}`
```
Comment (nguyentvan7) [inline src/PageDocumentation/index.tsx:206]: Use `charNames_gen.json` so we don't have to load every character's translation json

Comment (nguyentvan7) [inline src/PageDocumentation/index.tsx:206]: Also, can use Traveler's translated name? Just have it format as something like
`// Aether/Lumine`

Comment (nguyentvan7) [reply src/PageDocumentation/index.tsx:206]: While we are at it, we could also update the artifact and weapons to use `artifactNames_gen.json` and `weaponNames_gen.json`

**Review by nguyentvan7 — APPROVED** [review]

**Pull request merged**